image-copy-gcp/README.md

```diff
@@ -9,7 +9,7 @@ it like this:
 
 ```
 module "image-copy" {
-  source = "github.com/chainguard-dev/platform-examples//image-copy-gcp/iac"
+  source = "github.com/chainguard-dev/platform-examples/image-copy-gcp/iac"
```

Comment:
I think the double `//`s are expected, this is how you reference TF modules in directories in a GH repo.

Does it work without this?

Reply:
It _seemed_ to work fine but I took [a closer look at the docs](https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories) and see where the double slash comes from. Thanks for enlightening me!